Source/quests.cpp

```diff
@@ -72,7 +72,7 @@ int WaterDone;
 
 int qlist[MAXQUESTS];  ///< indices of quests to display in quest log window. `fistfinishedEntry` are active quests the rest are completed
 int qlistCnt;          ///< overall number of qlist entries
-int fistfinishedEntry; ///< first (nonselectable) finished quest in list

```

Comment:
![image](https://user-images.githubusercontent.com/14297035/129961096-7ecf0e2e-6d5e-4931-82f6-3485090e57e9.png)

Reply:
omg - and I thought you ment the casing 🤣